src/redux/reducers/movieReducer.js

```diff
@@ -0,0 +1,31 @@
+import {
+  RECEIVE_API_DATA,
+  CLICK_REQUEST_DATA,
+  REQUEST_API_DATA,
+} from '../actions';
+
+function movieReducer(state = {
+  inputs: '',
+
+}, action) {
+  switch (action.type) {
+    case REQUEST_API_DATA:
+      return {
+        ...state,
+        response: action.response,
```

Comment:
it would be nice to have smth specific, what response do you expect? please store data

Reply:
Updated the names